kll.py

```diff
@@ -1,820 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
Can restore this file if it is actually relevant - just thought it was more of a risk that it gets confused with the new code (I looked through it at first and was slightly confused)